#Alter optimisation cflags to strip out those already automatically specified
#by %%cmake rpm macro (which uses -o2) 
--- pcsx2-1.1.0/pcsx2/CMakeLists.txt
+++ pcsx2-1.1.0/pcsx2/CMakeLists.txt
@@ -32,45 +32,7 @@ set(DebugFlags
 	-g)
 
 # set optimization flags
-set(OptimizationFlags
-	-falign-functions
-	-falign-jumps
-	-falign-labels
-	-falign-loops
-	-fcaller-saves
-	-fcprop-registers
-	-fcrossjumping
-	-fcse-follow-jumps
-	-fcse-skip-blocks
-	-fdefer-pop
-	-fdelete-null-pointer-checks
-	-fgcse
-	-fgcse-lm
-	-fif-conversion
-	-fif-conversion2
-	-fmerge-constants
-	-foptimize-sibling-calls
-	-fpeephole2
-	-fregmove
-	-freorder-blocks
-	-freorder-functions 
-	-frerun-cse-after-loop
-	-fsched-interblock
-	-fsched-spec
-	-fstrict-overflow
-	-fthread-jumps
-	-ftree-ccp
-	-ftree-ch
-	-ftree-copyrename
-	-ftree-dce
-	-ftree-dominator-opts
-	-ftree-fre
-	-ftree-lrs
-	-ftree-pre
-	-ftree-sra
-	-ftree-ter
-	-ftree-vrp
-	-funit-at-a-time)
+set(OptimizationFlags)
 
 # Debug - Build
 if(CMAKE_BUILD_TYPE STREQUAL Debug)
